docs/using/config/configuration.rst

```diff
@@ -1,8 +1,8 @@
 .. _configuration:
 
-===================
-Configuring Flocker
-===================
+===============================
+Specifying your Flocker Cluster
```

Comment:
You're not specifing a flocker cluster. You're running some applications and creating some datasets/volumes.

The goal of the user here is not to do something to flocker, it's to run a database or whatever. Flocker is just a tool.